hera_mc/autocorrelations.py

```diff
@@ -27,8 +27,9 @@
 
 
 def _get_autos_from_redis(redishost=DEFAULT_REDIS_ADDRESS):
-
-    redis_pool = redis.ConnectionPool(host=DEFAULT_REDIS_ADDRESS)
+    if redishost is None:
+        redishost = DEFAULT_REDIS_ADDRESS
```

Comment:
Now that you aren't defaulting it to None, I don't think you need these lines.

Reply:
Fixed.